docs/source/index.rst

```diff
@@ -47,7 +47,7 @@ target environment.**
    Go <https://pkg.go.dev/github.com/apache/arrow/go>
    Java <java/index>
    JavaScript <js/index>
-   Julia <https://github.com/apache/arrow-julia/blob/main/README.md>
+   Julia <https://github.com/apache/arrow-julia>
```

Comment:
Could you explain why we should change to this URL?

Reply:
I'm sorry, it's link in version 7 docs, that's not valid anymore. Master is actually OK. Sorry to bother you.